### DISCUSSION TITLE
connect to the  search result from different/separate search box

### DISCUSSION THREAD

**Discussion by RPushpangadan** [microsoft-search/pnp-modern-search#2138] [Q&A]
Hi
I hope someone can help with my query. 

I would like to see if we can link PnP search results from a separate PnP search box which is on a different page? 

 currently, it's working when the search box and search results are on the same page. 

Appreciate it if I could get some help sorting this out.

Regards
Raj P

**Comment by wobba** [comment]
See https://microsoft-search.github.io/pnp-modern-search/usage/search-box/#configuration

**Comment by RPushpangadan** [comment]
Thank you Wobba, I have tried those suggestions and no luck with that.

here I have given the page URL to the new search page,  but when I do a search, its only redirects to the search page.



when we do the search from the home page search box, it will redirect to this page and I have to re-query again

![image](https://user-images.githubusercontent.com/91881356/172904278-b78be107-250a-4dfb-b919-e791fe078f83.png)

any help will be much appreciated.

Regards
Raj

**Reply by RPushpangadan** [reply]
Hi,  
can anyone let me know if there is a direct support page/forum for PnP?

**Reply by wobba** [reply]
This is it, and people respond and help when/if they can. But seems to be your setup is wrong somehow. You need to configure the search box on the landing page to consume the fragment or query param you are using. Then it should be populated.

**Reply by RPushpangadan** [reply]
Thanks, Wobba,  is it possible to provide a sample fragment or query pram?

**Reply by wobba** [reply]
Try using a query param instead. Like `q=` instead of a fragment and see if you have better luck.

**Comment by RPushpangadan** [comment]
Thanks Wobba

I am a small bit confused, as I am a complete beginner on PnP.

I used the query pram, the query is taken to the default search box( with no result), all we trying to do is link the two Pnp search boxes, so we should get the result with a single query. at the moment the home page PnP search box is taking to the other page and we have to query the search again to get the result.  I hope I am not confusing you.



![image](https://user-images.githubusercontent.com/91881356/174652500-9d11c93b-c3d5-452e-bd49-8980e0931459.png)

**Reply by wobba** [reply]
I suggest starting with the tutorials at https://microsoft-search.github.io/pnp-modern-search/scenarios/
